### DISCUSSION TITLE
Please make APK for my Xiaomi Mi Box S

### DISCUSSION THREAD

**Discussion by ipchesnokovaep** [giejay/Immich-Android-TV#38] [Q&A] [answered]
Hello!
Please make APK for my Xiaomi Mi Box S Android TV 9
I dont have enough skills for this

**Comment by faint069** [comment]
I have Xaomi Mibox S with android TV 9 and I just installed immich android tv from Google play Can't you do that?

**Comment by giejay** [comment, marked as answer]
Check the releases..they include the APK

**Reply by faint069** [reply]
> replies
All, except last one, 1.35 doesn't have apk
